.eslintrc.json

```diff
@@ -40,15 +40,19 @@
             "depConstraints": [
               {
                 "sourceTag": "scope:core",
-                "onlyDependOnLibsWithTags": ["scope:shared"]
+                "onlyDependOnLibsWithTags": ["scope:shared", "scope:utils"]
               },
               {
                 "sourceTag": "scope:shared",
-                "onlyDependOnLibsWithTags": ["scope:shared"]
+                "onlyDependOnLibsWithTags": ["scope:shared", "scope:utils"]
```

Comment:
I wasn't certain around scopes for now since shared is something we specifically use for payment integrations. So wasn't certain. Should I change it to shared 🤔